scripts/cli/base.cli.ts

```diff
@@ -5,11 +5,12 @@ import chalk from 'chalk';
 import * as commander from 'commander';
 import { compareVersions } from 'compare-versions';
 import * as fs from 'fs-extra';
```

Comment:
nit: doesn't look like we use any extras here vs the native `fs`

Reply:
It does provide some sugar methods, like `createFile` for creating empty files. I couldn't find a `touch` method in the native `fs` module but I could just write an empty file.